### PR TITLE
Updated module/extension version to 0.1.1

### PR DESCRIPTION
**Pull request by keithbabinec**
Note: do not merge until after #64 is merged.

### CODE REVIEW

File: vscode-extension/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"name": "azps-tools",
 	"displayName": "Azure PowerShell Tools",
 	"description": "(preview) Automate the migration of Azure PowerShell scripts and modules from AzureRM to the Az PowerShell module.",
-	"version": "0.1.0",
+	"version": "0.1.1",
```
Comment (dingmeng-xue) [inline vscode-extension/package.json:5]: @chenlei-yang , please check if extension version needs to be updated?

**Review by dingmeng-xue — COMMENTED** [review]
@chenlei-yang please review this one.

File: vscode-extension/package.json (hunk 1 above)
Comment (chenlei-yang) [reply vscode-extension/package.json:5]: @dingmeng-xue Yes, we need to change the target version in the language server, and I already committed it.

**Comment by dcaro** [comment]
@dingmeng-xue @chenlei-yang can you confirm if this PR still valid?

**Comment by keithbabinec** [comment]
Closing this -- now obsolete with the other PR that pushed things to 1.0

**Pull request closed without merging**